redash/query_runner/druid.py

```diff
@@ -0,0 +1,93 @@
+try:
+    from pydruid.db import connect
+    enabled = True
+except ImportError:
+    enabled = False
+
+from redash.query_runner import BaseQueryRunner, register
+from redash.utils import json_dumps, json_loads
+
+TYPES_MAP = {
+}
+
+
+class Druid(BaseQueryRunner):
+    noop_query = "SELECT 1"
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "host": {
+                    "type": "string",
+                    "default": "localhost"
+                },
+                "port": {
+                    "type": "number",
+                    "default": 8082
+                },
+                "scheme": {
+                    "type": "string",
+                    "default": "http"
+                }
+            },
+            "order": ['scheme', 'host', 'port'],
+            "required": ['host']
```

Comment:
No username/password support?

Reply:
While there seems to be [support for basic auth](https://github.com/druid-io/pydruid/blob/master/pydruid/client.py#L40) in the client, the docs for Druid's "experimental" SQL dialect don't show anything regarding authentication. Perhaps they expect people to protect endpoints by other means?

Reply:
👍 OK, let's just keep as is and if someone asks for it, revisit.